package.json

```diff
@@ -5,6 +5,7 @@
   "dependencies": {
     "baltimorecounty-react-components": "0.0.15",
     "date-fns": "^1.29.0",
+    "moment": "^2.24.0",
```

Comment:
it doesn't look like you are suing this anywhere? can you please remove the dependency?

Reply:
moment is removed